src/NotificationBar/NotificationBar.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { BarDiv } from './NotificationBar.styles';
+
+export type Props = {
+  children?: React.ReactElement;
+  variant?: 'alert' | 'refresh';
```

Comment:
can we make this `'alert' | 'info'` to keep it more in line with convention and more reusable?